### DISCUSSION TITLE
Workflow for handling track changes and comments from exported docx back to zettlr

### DISCUSSION THREAD

**Discussion by menelic** [Zettlr/Zettlr#3337] [Q&A]
Zettlr is great for almost every part of the academic workflow - with the exception of reviews and revisions, it seems. I have done quite a bit of research but have not found a usable workflow with two way conversion of track changes and comments.  The documentation states:

>Even after switching to Markdown, you will have to deal with Word-documents and need to share your thoughts with people who do not use Markdown.

But there seems to be a lack of documentation or workflow examples on how to deal with track changes or comments made in a text that has been converted to docx to share with collaborators or reviewers. I am aware of #728 but am wondering of @nathanlesage or others have a workflow example on how to go about this in the meantime. There might be some ways or workflows some of you have devised to make sure writing can continue in zettlr even after having received revisions in docx? I believe having some solutions documented would make it a lot easier for knowledge workers esp in academia to adopt zettlr as a writing tool

**Comment by nathanlesage** [comment]
I honestly are just sending out PDF files to people so that I can manually look at the changes, or just spin up Word (not completely off the hook with that, unfortunately).

I don't know to what extent that is possible at all, because we do rely on Pandoc for this, and as far as I'm concerned you can just accept or reject all changes upon import, but I may be wrong.

Would be a cool feature, though

**Reply by menelic** [reply]
Pandoc can handle inserts, deletions and comments from Word - as per the pandoc documentation:

> --track-changes=accept|reject|all

> Specifies what to do with insertions, deletions, and comments produced by the MS Word “Track Changes” feature. accept (the default) processes all the insertions and deletions. reject ignores them. Both accept and reject ignore comments. all includes all insertions, deletions, and comments, wrapped in spans with insertion, deletion, comment-start, and comment-end classes, respectively. The author and time of change is included. all is useful for scripting: only accepting changes from a certain reviewer, say, or before a certain date. If a paragraph is inserted or deleted, track-changes=all produces a span with the class paragraph-insertion/paragraph-deletion before the affected paragraph break. This option only affects the docx reader.


if zettlr could implement this via https://github.com/ickc/pancritic or another implemetation of criticmarkup this would not just allow for round trip ex and import form word or odt, but also create feature parity with Word and LibreOffice regarding a key functionality for academic writing. This is key for co-authoring, revisions and other collaborative knowledge projects as well as for processing journal revisions.

see also relevant and current discussion here https://github.com/jgm/pandoc/issues/2873
